### DISCUSSION TITLE
Long delay after entering a command

### DISCUSSION THREAD

**Discussion by amirmasoudabdol** [starship/starship#1555] [Q&A] [answered]
I've just installed the starship on a server that I'm using. While the installation went well, there is a massive delay after the execution of each command. I basically have to wait somewhat around 10sec before I have the command line again. I'm running starship on Debian with bash. Since I don't have the root access I installed it in `/usr/local/bin` and link to it in the `.bashrc`.

Does anyone have any ideas why this is happening? Or have experienced something similar?

**Comment by matchai** [comment, marked as answer]
Does this happen in a git repo, or everywhere in the system?
#940 and #1469 have fixes planned, but are both git-specific.

**Reply by amirmasoudabdol** [reply]
It's been a while that I've reported it, and I think it's resolved as I haven't noticed it anymore! Thanks!
